biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -188,7 +188,7 @@ The Business Activity Monitoring Tools include:
 
 BAM alerts require BAM tools to be enabled.
 
-- BAM Alerts service requires SQL_AS_ADOMD client to run. Install SQL Server 2016 Analysis Services ADOMD from https://www.microsoft.com/en-us/download/details.aspx?id=52676 .
+- BAM Alerts service requires SQL_AS_ADOMD client to run. Install SQL Server 2016 Analysis Services ADOMD from [SQL Server 2016 Feature Pack](https://www.microsoft.com/en-us/download/details.aspx?id=52676).
```

Comment:
do we have a dependency on EN version of ADOMD client? if not, please remove 'en-us' from the link